### DISCUSSION TITLE
is it possible to give multiple password urls?

### DISCUSSION THREAD

**Discussion by isshwar** [ltb-project/self-service-password#734] [Q&A]
Hi,

I have ssp running internally on kubernetes sitting behind a reverseproxy and only accessible from internal ip/hosts .I have external users wanting to access it. I have added a location /ssp directive in one of our nginx rp config to proxy_pas to our ssp service, so the requests from external users are considered internal.

Now, the issue is while the users are able to request the link for password reset, but the url is again of the internal domain not the one external users can access.

so, my question is ,is it possible to send multiple url's one hard code with the external domain name and the other with internal domain name? 

Thanks

**Comment by coudot** [comment]
You can configure the `$reset_url` : https://self-service-password.readthedocs.io/en/latest/config_tokens.html#reset-url

As this is a PHP expression, you can easily code what value you want to return.

**Comment by coudot** [comment]
Not that you can also have a multi-tenant configuration: https://self-service-password.readthedocs.io/en/latest/config_general.html#multi-tenancy

**Comment by isshwar** [comment]
Thanks for your reply.

Can I have multiple values there?

**Reply by coudot** [reply]
You can define just one value for this parameter, but this value can be a PHP expression that will let you set the correct URL depending on the context.
